kuksa_databroker/databroker/src/broker.rs

```diff
@@ -1340,32 +1487,78 @@ impl<'a, 'b> AuthorizedAccess<'a, 'b> {
 
     pub async fn subscribe(
         &self,
-        valid_entries: HashMap<i32, HashSet<Field>>,
+        valid_entries: HashMap<i32, (HashSet<Field>, types::ChangeType)>,
```

Comment:
Why is `ChangeType` added as an argument here? Each signal already has a `ChangeType` associated with it, so why should the caller of `subscribe` supply this information?

What would be a relevant argument to add here though, is at what frequency the caller wants continuous signals to be updated at. And this would only be required for signals of type `ChangeType::Continuous`.

Reply:
I had to choose between changing the subscribe method parameters to pass the entire entry and change the method implementation so I went for the option that makes less changes in the code by only adding the change type info.
About the frequency I do agree, I dont if I should include it in this PR or make a new one for it since I would have to change the proto file to include the frequency param.

Reply:
I'm pretty sure you don't need to change the signature at all as you have full access to the db in the `subscribe` function (and hence can get the change_type given an id).